README.md

```diff
@@ -59,8 +63,17 @@ func main() {
 
 	c.SubscribeTopics([]string{"myTopic", "^aRegex.*[Tt]opic"}, nil)
 
-	for {
-		msg, err := c.ReadMessage(-1)
+	sig := make(chan os.Signal, 1)
+	// get notified by signal when interrupted and break the consuming loop
+	signal.Notify(sig, os.Interrupt)
+	closed := false
+	go func() {
+		<-sig
+		closed = true
+	}()
+
+	for !closed {
+		msg, err := c.ReadMessage(100 * time.Millisecond)
 		if err == nil {
 			fmt.Printf("Message on %s: %s\n", msg.TopicPartition, string(msg.Value))
 		} else {
```

Comment:
I tested this and realized we should probably change this else clause to something like

`else if err.(kafka.Error).Code() != kafka.ErrTimedOut { ... }`

Reason for this is, by making ReadMessage have a timeout, the console will be filled with a ton of timeout messages